rpmfusion-free-release.spec

```diff
@@ -75,21 +79,30 @@ ln -s $(basename %{SOURCE28}) %{buildroot}%{_sysconfdir}/pki/rpm-gpg/RPM-GPG-KEY
 
 
 # Yum .repo files
-%{__install} -p -m644 %{SOURCE1} %{SOURCE2} %{SOURCE3} %{SOURCE4} \
+%{__install} -p -m644 %{SOURCE1} %{SOURCE2} %{SOURCE3} %{SOURCE4} %{SOURCE5} \
     %{buildroot}%{_sysconfdir}/yum.repos.d
 
+%triggerpostun -- livna-release
+sed -i '/\[rpmfusion-free-tainted\]/,/^ *\[/ s/enabled=0/enabled=1/' \
+    /etc/yum.repos.d/rpmfusion-free-tainted.repo
```

Comment:
Looks good, did you tested it ?

Reply:
should probably ends with || : to avoid any random issue breaking the rpm transaction